src/componants/ViewSingleArticle.jsx

```diff
@@ -13,11 +15,25 @@ const ViewSingleArticle = () =>
         .then((results) => setArticleBody(results.data.data[0]))
      }, [])
 
+     useEffect(() => {
+        getSingleComments(article_ID)
+        .then((results) => setArticleComments(results.data.comments))
+
+     }, [])
+
+     console.log(articleComments)
+
     return <>
     <h1>{articleBody.title}</h1>
     <h3>Written by {articleBody.author}</h3>
     <img src={articleBody.article_img_url} alt="article picture" />
     <p>{articleBody.body}</p> 
+    <h3 > View comments</h3>
```

Comment:
The text of this makes me think it's a button, maybe that's something you are going to implement later, but if not I would rename to Comments